templates/kubernetes-cluster.template

```diff
@@ -381,6 +383,36 @@ Resources:
             # so pods can communicate
             kubectl apply -f https://${QSS3BucketName}.s3.amazonaws.com/${QSS3KeyPrefix}/scripts/${NetworkingProvider}.yaml
 
+            # Expose the api server over ELB
+            kubectl -n kube-system expose pod \
+              kube-apiserver-$(hostname -f) \
+              --type=LoadBalancer \
+              --port 443 \
+              --target-port=6443 \
+              --name apiserver-public
+
+            # Wait until it's available
+            set +e
```

Comment:
That works -- you can also put the assignment in an if statement and it won't trigger the errexit, I think.